gef-ui/server/models/application.js

```diff
@@ -127,10 +127,10 @@ class Application {
       }
 
       return application;
-    } catch (err) {
+    } catch (error) {
       // eslint-disable-next-line no-console
```

Comment:
do we need this since it's console.error

Reply:
Same as above, but no we don't.